edx_django_utils/plugins/plugin_contexts.py

```diff
@@ -19,8 +19,10 @@ def get_plugins_view_context(project_type, view_name, existing_context=None):
     selected function to get their context dicts.
 
     Params:
-        project_type: a string that determines which project (lms or studio) the view is being called in. See the
-            ProjectType enum in plugins/constants.py for valid options
+        project_type: a string that determines which project the view is being called in. See the
+            ProjectType enum in
+            https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/plugins/constants.py#L14 for
```

Comment:
1. Also make this a permalink.
2. It is strange that there isn't an equivalent doc needed somewhere for the SettingsType.